iOS/DuckDuckGo/DebugScreensViewModel+Screens.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  DebugScreensViewModel+Screens.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import SwiftUI
+import UIKit
+import WebKit
+import BareBonesBrowserKit
+import Core
+
+extension DebugScreensViewModel {
+
+    /// Just add your view or debug building logic to this array. In the UI this will be ordered by the title.
+    /// Note that the storyboard is not passed to the controller builder - ideally we'll mirgate away from that to SwiftUI entirely
+    var screens: [DebugScreen] {
+        return [
+            // MARK: Actions
+            .action(title: "Reset Autoconsent Prompt", { _ in
+                AppUserDefaults().clearAutoconsentUserSetting()
+            }),
+            .action(title: "Reset Sync Promos", { d in
+                let syncPromoPresenter = SyncPromoManager(syncService: d.syncService)
+                syncPromoPresenter.resetPromos()
+            }),
+            .action(title: "Reset TipTik", { d in
```

Comment:
```suggestion
            .action(title: "Reset TipKit", { d in
```

Reply:
Thanks! Would have used your suggestion but the GitHub UI is broke and I couldn't see the button.